ts/components/NavBarLabel.tsx

```diff
@@ -70,10 +71,10 @@ const NavBarLabel: React.FunctionComponent<Props> = (props: Props) => {
           color: options.tintColor === null ? undefined : options.tintColor
         }
       ]}
-      accessibilityLabel={I18n.t("navigation.accessibility", {
+      accessibilityLabel={`${isSelected}, ${I18n.t("navigation.accessibility", {
```

Comment:
```suggestion
      accessibilityLabel={`${isSelected}${I18n.t("navigation.accessibility", {
```
Following the above suggestion.

Reply:
Fix PR has suggested ✅